docs/config.md

```diff
@@ -46,3 +46,4 @@ Each service description is an object with the keys:
 
  * `authority` _String_. The domain name which the annotation service is associated with.
  * `grantToken` _String|null_. An OAuth grant token which the client can exchange for an access token in order to make authenticated requests to the service. If _null_, the user will only be able to read rather than create or modify annotations. (Default: _null_)
+ * `icon` _String|null_. The path to an image for the annotation service. This image will appear to the left of the name of the currently selected group. The recommended size for the image is 16x16 and format is SVG.
```

Comment:
You mean _URL_ rather than _path_, right? Also, since SVG icons are vectors they don't have a _size_ as such. What you could say is that it _should be suitable for display at 16x16px._